pkgs/os-specific/linux/mdevd/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, skawarePackages }:
+
+with skawarePackages;
+
+buildPackage {
+  pname = "mdevd";
+  version = "0.1.3.0";
+  sha256 = "0spvw27xxd0m6j8bl8xysmgsx18fl769smr6dsh25s2d5h3sp2dy";
+
+  description = "mdev-compatible Linux hotplug manager daemon";
+  platforms = lib.platforms.linux;
+
+  outputs = [ "bin" "out" "dev" "doc" ];
```

Comment:
I'm a bit confused what happens to mdevd's libraries. Currently the `out` output comes out empty although by default it should install at least a static library, right? I'm guessing `out` is where you intended libraries to go.

Also currently `nix-support` ends up in `dev` instead of `out` which is a bit weird.

Reply:
> I'm a bit confused what happens to mdevd's libraries. Currently the out output comes out empty although by default it should install at least a static library, right? I'm guessing out is where you intended libraries to go.

What libraries would you expect it to install?

mdevd's make install only wants to install /bin/mdevd, /bin/mdevd-coldplug, and /usr/include/mdevd/config.h.

The empty out output is a little strange but that seems to be how other skaware packages work -- for example it's the same in s6-rc.

> Also currently nix-support ends up in dev instead of out which is a bit weird.

This appears to be intentional:

https://github.com/NixOS/nixpkgs/blob/1421404d425ec3af31f9d3f53a69fd94b1b88657/pkgs/build-support/setup-hooks/multiple-outputs.sh#L167

Reply:
Ah yeah, makes sense, was just a bit confused by the empty output since I expected at least nix-support to end up there and the generic Makefile of mdevd.